modules/tasks/tasks/src/main/java/org/motechproject/tasks/json/TaskDeserializer.java

```diff
@@ -93,4 +108,27 @@ private void setProperty(String propertyName, String jsonPropertyName, JavaType
             }
         }
     }
+
+    private String removeSpareQuotation(String input) {
```

Comment:
this really seems suspect to me, why do we have spare quotations in the first place? Moreover, this doesn't remove quotation, it removes the first and last char, whatever they are.